frameworks/Python/flask/app-contrast.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+from functools import partial
+import json
+from operator import attrgetter
+import os
+from random import randint
+import sys
+from email.utils import formatdate
+
+import flask
+from flask import Flask, request, render_template, make_response, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import create_engine
+from sqlalchemy.ext import baked
+
+if sys.version_info[0] == 3:
+    xrange = range
+
+_is_pypy = hasattr(sys, 'pypy_version_info')
+
+DBDRIVER = 'mysql+pymysql' if _is_pypy else 'mysql'  # mysqlclient is slow on PyPy
+DBHOST = 'tfb-database'
+
+
+# setup
+
+app = Flask(__name__)
+
+# Start Contrast Specific Changes
+from contrast.agent.middlewares.flask_middleware import FlaskMiddleware as ContrastMiddleware
+app.wsgi_app = ContrastMiddleware(app)
+# End Contrast Specific Changes
+
+app.config['SQLALCHEMY_DATABASE_URI'] = DBDRIVER + '://benchmarkdbuser:benchmarkdbpass@%s:3306/hello_world?charset=utf8' % DBHOST
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['JSONIFY_PRETTYPRINT_REGULAR'] = False
+db = SQLAlchemy(app)
+dbraw_engine = create_engine(app.config['SQLALCHEMY_DATABASE_URI'], connect_args={'autocommit': True}, pool_reset_on_return=None)
+
+bakery = baked.bakery()
+
+
+# models
+
+class World(db.Model):
+    __tablename__ = "world"
+    id = db.Column(db.Integer, primary_key=True)
+    randomNumber = db.Column(db.Integer)
+
+    # http://stackoverflow.com/questions/7102754/jsonify-a-sqlalchemy-result-set-in-flask
+    @property
+    def serialize(self):
+        """Return object data in easily serializeable format"""
+        return {
+            'id'         : self.id,
+            'randomNumber': self.randomNumber
+        }
+
+    @staticmethod
+    def get(ident):
+        baked_query = bakery(lambda s: s.query(World))
+        return baked_query(db.session()).get(ident)
+
+
+class Fortune(db.Model):
+    __tablename__ = "fortune"
+    id = db.Column(db.Integer, primary_key=True)
+    message = db.Column(db.String)
+
+
+# views
+
+# flask.jsonify doesn't allow array at top level for security concern.
+# So we should have oriiginal one.
```

Comment:
Nit, but oriiginal -> original

Reply:
That's a typo in what I copied over, so I'm going to leave it.